libc/src/sys/ioctl/linux/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_entrypoint_object(
+  ioctl
+  SRCS
+    ioctl.cpp
+  HDRS
+    ../ioctl.h
+  DEPENDS
+    libc.include.sys_ioctl
```

Comment:
I think whats going wrong in the presubmit test failure is that you did not provide a definition of libc.include.sys_ioctl and we don't have an existing one, yet you are adding a dependency on it.

I'll bet you need a `add_gen_header` in libc/include/CMakeLists.txt for this.  Checkout how it's done for sys/select.h.def as an example.

Reply:
ah, no that's not it. nvm